qa/tasks/cephfs/kernel_mount.py

```diff
@@ -68,7 +68,10 @@ def mount(self, mntopts=None, check_status=True, **kwargs):
                 self.enable_dynamic_debug()
             self.ctx[f'kmount_count.{self.client_remote.hostname}'] = kmount_count + 1
 
-        self.gather_mount_info()
+        try:
+            self.gather_mount_info()
+        except:
+            log.warn('failed to fetch mount info - tests depending on mount addr/inst may fail!')
```

Comment:
Maybe it's better to add similar check to underlying methods instead, so that exception message can be more specific. A more specific exception message will be make diagnosing problem (from logs) easier.